.appveyor.yml

```diff
@@ -0,0 +1,68 @@
+os: Visual Studio 2015
+platform: x64
+environment:
+  global:
+    MINICONDA: "C:\\Miniconda36-x64"
+    REPO_DIR: httpstan
+    PACKAGE_NAME: httpstan
+    BUILD_COMMIT: 0.9.0
+    WHEELHOUSE_UPLOADER_USERNAME: "stan-worker"
+    WHEELHOUSE_UPLOADER_REGION: "IAD"
+    WHEELHOUSE_UPLOADER_SECRET:
+      secure: 9xnd08h7pp+3ytu6xKKwdo/fT9VXS7GjF6LkgLGynZqWQmZZ1PHDDD2epJa1HJbz
+  matrix:
+    - PYTHON_VERSION: 3.6
+    - PYTHON_VERSION: 3.7
+
+install:
+  # check out the desired commit
+  - cd %REPO_DIR%
+  - git fetch origin
+  - git checkout %BUILD_COMMIT%
```

Comment:
not sure if it's powershell that fails

this could work

    - git checkout -q %BUILD_COMMIT%

https://github.com/dahlbyk/posh-git/issues/109#issuecomment-68620271

Reply:
I believe it was because I was missing the submodule checkout.